hugegraph-dist/pom.xml

```diff
@@ -179,10 +179,11 @@
                         </goals>
                         <configuration>
                             <target>
-                                <tar destfile="${final.destfile}"
-                                     basedir="${top.level.dir}"  
-                                     includes="${final.name}/**"
-                                     compression="gzip"/>
+                                <tar destfile="${final.destfile}" compression="gzip">
+                                    <tarfileset dir="${top.level.dir}/" filemode="755">
```

Comment:
we do need a test to ensure these problems

Reply:
I have tested, the permission problem is fixed